packages/eslint-plugin/docs/rules/consistent-type-assertions.md

```diff
@@ -0,0 +1,77 @@
+# Enforces consistent usage of type assertions. (consistent-type-assertions)
+
+## Rule Details
+
+This rule aims to standardise the use of type assertion style across the codebase.
+
```

Comment:
You start out consistently using "type assertions" (the more official term), but then use "cast" (the commonly used, but less strictly accurate term) a few times as well.

Please could you add a short explanation in the intro about this fact/relationship?

e.g. something like:

> Type assertions are also commonly referred as "type casting" in TypeScript (even though it is technically slightly different to what is understood by type casting in other languages), so you can think of type assertions and type casting referring to the same thing. It is essentially you saying to the TypeScript compiler, "in this case, I know better than you!"